config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -1,3 +1,37 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-make-test
+    decorate: true
+    cluster: k8s-infra-prow-build
+    skip_branches:
+    - release-\d+.\d+ # per-release job
+    labels:
+      preset-service-account: "true"
+    optional: true
+    always_run: false
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20210223-952586a143-master
+          command:
+            - runner.sh
+            - bash
+          args:
+            - -c
+            - |
+              # Restore the cache
+              time curl https://storage.googleapis.com/kubernetes-jenkins/cache/poc/k8s-test-cache.tar.gz -o cache.tar.gz
+              mkdir -p _output/local/go/
+              time tar -xzf cache.tar.gz -C _output/local/go
+              # Run tests as usual
+              time make test KUBE_RACE=-race KUBE_TIMEOUT=--timeout=240s
```

Comment:
TODO: what timeout do we really want here? This one is observed from GKE's equivalent infra, and is at least lower than the 600s below.
TODO: should `KUBE_RACE=-race` be default? In bazel we just had race by default IIRC. Ideally locally I should just `make test` and it should do the preferred thing ... I don't know why it isn't default yet though.